configure.in

```diff
@@ -7807,7 +7823,7 @@ EXTRALIBS_HTML="$MSPACK_LINK"
 EXTRALIBS_MEDIA="$GST_LIBS"
 EXTRALIBS_STC="-lwxscintilla${WX_LIB_FLAVOUR}-${WX_RELEASE}${HOST_SUFFIX}"
 if test "$wxUSE_GUI" = "yes"; then
-    EXTRALIBS_GUI=`echo $GUI_TK_LIBRARY $PNG_LINK $JPEG_LINK $TIFF_LINK $WEBKIT_LINK`
+    EXTRALIBS_GUI=`echo $GUI_TK_LIBRARY $PNG_LINK $JPEG_LINK $LZMA_LINK $JBIG_LINK $TIFF_LINK $WEBKIT_LINK`
```

Comment:
This is unlikely to happen on Linux systems, but if we were linking statically, the new additions ought to come after `TIFF_LINK` which depends on them.

Reply:
Yeah, but `-llzma` and `-ljbig` aren't added anywhere to `TIFF_LINK` or am I missing something?

Reply:
Sorry for being unclear. What I meant was just that linking with `-llzma -ljbig -ltiff` was going to fail when using static libraries. You need to link with `-ltiff -llzma -ljbig` to make it work, order matters for static libraries.

Reply:
Oh, I didn't know about the order, thanks. Btw I'm using static linking on Windows, and there weren't any errors. I can change the order.

Reply:
Yes, the order doesn't matter under MSW, only for traditional Unix linkers, such as all the usual ELF ones (GNU ld, gold, etc).